Content.Client/CriminalRecords/CriminalRecordsConsoleWindow.xaml.cs

```diff
@@ -219,7 +219,8 @@ private void FilterListingOfRecords(string text = "")
 
     private void SetStatus(SecurityStatus status)
     {
-        if (status == SecurityStatus.Wanted || status == SecurityStatus.Suspected)
+        if (status == SecurityStatus.Wanted || status == SecurityStatus.Suspected || status == SecurityStatus.Demote ||
```

Comment:
Создай сверху метода вот такой список, где будут нужные статусы

```
private HashSet<SecurityStatus> AllowedStatusList = new {Хуячишь, сюда, все, статусы}
```

Затем на этой строчке

```suggestion
        if (AllowedStatusList.Contains(stats)
```

Мб неточно написал, но +- оно так и должно  быть